src/catalog/postgres/pg_core_impl.cpp

```diff
@@ -82,8 +82,9 @@ void PgCoreImpl::BootstrapPRIsPgAttribute() {
   pg_attribute_all_cols_prm_ = columns_->ProjectionMapForOids(pg_attribute_all_oids);
 
   const std::vector<col_oid_t> get_columns_oids{PgAttribute::ATTNUM.oid_,     PgAttribute::ATTNAME.oid_,
-                                                PgAttribute::ATTTYPID.oid_,   PgAttribute::ATTLEN.oid_,
+                                                PgAttribute::ATTTYPID.oid_,   PgAttribute::ATTTYPMOD.oid_,
                                                 PgAttribute::ATTNOTNULL.oid_, PgAttribute::ADSRC.oid_};
+  // TODO(Matt): Maybe length should still be read back out from the table rather than relying ton Column's constructor?
```

Comment:
`VAR_ARRAY` will throw a runtime exception if we try and get the size (since there's no case in `TypeUtil::GetTypeSize`). I don't think that's an issue because none of our columns in pg_attribute are an array type, but postgres does have a couple of columns in pg_attributes that have array types (https://www.postgresql.org/docs/9.3/catalog-pg-attribute.html: attacl, attoptions, attfdwoptions)

Probably doesn't matter, but I thought I'd point it out.